src/block/types.js

```diff
@@ -334,7 +334,7 @@ export type BlockPropsT = {
   placeItems?: ResponsiveT<string>,
   /** available values: https://developer.mozilla.org/en-US/docs/Web/CSS/place-self */
   placeSelf?: ResponsiveT<string>,
-  flexWrap?: ResponsiveT<boolean>,
+  flexWrap?: ResponsiveT<string | boolean>,
```

Comment:
I would reference the block styled component. this prop should be pinned to boolean if any provided string will coerce the result to `wrap`